tsconfig.production.json

```diff
@@ -4,7 +4,7 @@
     "module": "commonjs",
     "target": "es2015",
     "declaration": true,
-    "outDir": "./dist",
+    "outDir": "./build",
     "allowJs": true,
     "jsx": "preserve",
     "skipLibCheck": true,
```

Comment:
If the `tsconfig.json` file is getting removed, would it make sense to rename this file to `tsconfig.json`?